doc/PythonSupport.md

```diff
@@ -45,10 +45,14 @@ Here are several noteworthy modules we haven't supported yet:
 - `curses` - not supported
 - `tkinter` - not supported
 
-**Caveat**: Any third-party Python library that calls into the unsupported
+**Caveat**:
+
+* Any third-party Python library that calls into the unsupported
 standard Python APIs are not supported neither. Sometimes they can produce
 unpredictable errors which are tricky to root cause.
 
+- Locale support - musl libc [has very limited locale support](https://wiki.musl-libc.org/open-issues.html). Python program running in Mystikos would suffer the same limitation, since Mystikos is using musl.
```

Comment:
Can you please give an example of how does the limited locale support affect a Python program? For example, as a Python developer, I might not know about MUSL or locale. What I care about is what kind of Python library or API calls would fail in Mystikos because of locale support.